static/styles/popovers.css

```diff
@@ -707,5 +707,11 @@ ul {
             margin: 0 5px 5px -10px;
             text-indent: 10px;
         }
+
+        .dropdown-menu {
+            position: fixed;
+            top: 125px;
+            left: 40px;
```

Comment:
I guess these numbers are going to need to change every time we tweak the design of the modal?